src/components/LinearProgressBar/LinearProgressBar.driver.ts

```diff
@@ -0,0 +1,32 @@
+import {ComponentFactory} from 'wix-ui-test-utils/driver-factory';
+import {tooltipDriverFactory} from 'wix-ui-core/dist/src/components/Tooltip/Tooltip.driver';
+import {
+  linearProgressBarDriverFactory as coreLinearProgressBarDriverFactory,
+  LinearProgressBarDriver as CoreLinearProgressBarDriver } from 'wix-ui-core/dist/src/components/LinearProgressBar/LinearProgressBar.driver';
+import {BaseDriver, DriverFactory} from 'wix-ui-test-utils/driver-factory';
+
+export interface LinearProgressBarDriver extends CoreLinearProgressBarDriver {
+    /* Returns true if the tooltip is shown */
+    isTooltipShown: () => boolean;
+    /* Returns true if the error icon is shown */
+    isErrorIconShown: () => boolean;
+    /* Returns true if the success icon is shown */
+    isSuccessIconShown: () => boolean;
+    /* Perform hover on tooltip */
+    hoverOnTooltip: () => void;
+}
+
+export const linearProgressBarDriverFactory: DriverFactory<LinearProgressBarDriver> = ({ element, eventTrigger, wrapper }: ComponentFactory): LinearProgressBarDriver => {
+    const tooltipDriver = tooltipDriverFactory({element: element.querySelector(`[data-hook='tooltip']`), wrapper, eventTrigger});
+    const coreProgressBarDriver = coreLinearProgressBarDriverFactory({element, wrapper, eventTrigger});
+    const errorIcon = () => element.querySelector(`[data-hook='error-icon']`);
+    const successIcon = () => element.querySelector(`[data-hook='success-icon']`);
+
+    return {
+        ...coreProgressBarDriver,
+        isTooltipShown: () => tooltipDriver.isContentElementExists(),
+        hoverOnTooltip: () => tooltipDriver.mouseEnter(),
```

Comment:
If someone woult want to do some more stuff with the tooltip like getting the error message, I suggest having `getTooltip()` method that will return the tooltip driver as a composition of drivers.

Then instead of doing `driver. hoverOnTooltip()` you would do `driver.getTooltip(). mouseEnter()` which is ok.